{{ cookiecutter.package_name }}/tox.ini

```diff
@@ -1,13 +1,24 @@
 [tox]
 envlist =
-    py{36,37,38}-test
+    py{36,37,38}
     build_docs
     codestyle
 isolated_build = true
 # This is included for testing of the template. You can remove it safely.
 skip_missing_interpreters = True
+requires =
+    setuptools >= 30.3.0
+    pip >= 19.3.1
+    tox-pypi-filter >= 0.10
 
 [testenv]
+
+# The following option combined with the use of the tox-pypi-filter above allows
+# project-wide pinning of dependencies, e.g. if new versions of pytest do not
+# work correctly with pytest-astropy plugins. Most of the time the pinnings file
+# should be empty.
+pypi_filter_requirements = https://raw.githubusercontent.com/sunpy/package_template/master/sunpy_version_pins.txt
```

Comment:
We want this pinned in the package_template repo?

Reply:
You mean why have I put this file here?

I put it here because it feels like everyone using this repo would be using it. We could also put it in core?

Reply:
It just seems a bit odd to me that if we have to pin a test dep, we have to go to this repo. That's it.

Reply:
Fair, put it in core?

Reply:
I wonder if core makes any more sense. I don't really have any bright ideas at the moment.

Reply:
Will open an issue.